### ISSUE TITLE
Tag releases more often!

### ISSUE THREAD

**Issue by jurchiks** [mk-j/PHP_XLSXWriter#267]
I've recently stumbled upon the PHP 7.4 notice about string index access via curly braces, and it has no version, which is a bummer.
I see that your latest version is from January 2018, which makes it over 2 years old, even though there is plenty of changes done since then. That's not a good way to get users to adopt your library, because almost nobody cares about the master branch.

**Comment by TeeJay-net** [comment]
@mk-j yes please, this is very important now that PHP 7.4 is being adopted.

**Comment by mk-j** [comment]
I just tagged one.
